assets/scss/_util.scss

```diff
@@ -0,0 +1,39 @@
+@for $i from 10 through 60 {
+  @if $i % 5 == 0 {
+    .mb-#{$i} {
+      margin-bottom: #{$i}px;
+    }
+
+    .mt-#{$i} {
+      margin-top: #{$i}px;
+    }
+
+    .pb-#{$i} {
+      padding-bottom: #{$i}px;
+    }
+
+    .pt-#{$i} {
+      padding-top: #{$i}px;
+    }
+  }
+}
```

Comment:
Bootstrap has defined these already: https://getbootstrap.com/docs/4.4/layout/utilities-for-layout/

Why do we want to define these ourselves ?

Reply:
bootstrap's margin/padding util classes are limited to 6 steps and not in steps 5-10-15-20-ect.